debug_toolbar/panels/sql/tracking.py

```diff
@@ -84,7 +84,12 @@ def __init__(self, cursor, db, logger):
 
     def _quote_expr(self, element):
         if isinstance(element, six.string_types):
-            return "'%s'" % force_text(element).replace("'", "''")
+            try:
+                return "'%s'" % force_text(element).replace("'", "''")
+            except DjangoUnicodeDecodeError:
+                return repr(element)
+        elif isinstance(element, six.binary_type):
```

Comment:
If you put this check first, do you still need the `except DjangoUnicodeDecodeError`? I guess it is required for Python 2.7?